.quickstart/quickstart.yml

```diff
@@ -0,0 +1,19 @@
+database_key: marketo_database
+schema_key: marketo_schema
+
+dbt_versions: ">=1.3.0 <2.0.0"
+
+table_variables:
+  marketo__enable_campaigns:
+    - program
```

Comment:
```suggestion

```

Reply:
@fivetran-reneeli program is intentionally included the variable is used in the [program staging model](https://github.com/fivetran/dbt_marketo_source/blob/e3480427b2067ff52ad9133fa4ec779d56e71d11/models/stg_marketo__program.sql#L1).

Reply:
Ah thanks for pointing me to that 👍